docs/site/tutorials/core/1-introduction.md

```diff
@@ -0,0 +1,80 @@
+---
+lang: en
+title: 'Introduction of the application scenario'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/core-tutorial-part1.html
+---
+
+# Introduction
+
+At the core of LoopBack 4, we provide a powerful Inversion of Control and
+Dependency Injection container in TypeScript that serves as the foundation to
```

Comment:
The links to IoC and DI are missing.

Reply:
Do you want to add the ones from wikipedia or LB docs?